css/index.css

```diff
@@ -176,6 +176,7 @@ header nav .links {
 header nav a {
   color: black;
   text-decoration: none;
+  transition: color 0.5s ease;
```

Comment:
Good job at adding animation. You could try to do something more fancier next time (have a button to bump when hovered, an image to fade in or something to rotate for instance).
Check out the GreenSock link I sent to you.